docs/source/tutorial.rst

```diff
@@ -63,6 +63,19 @@ Or choose a model from a pre-loaded library, like ``Modelica Standard Library``:
 
 This will return an instance of a ``Model`` class object.
 
+Setting logging levels for analysis
+-----------------------------------
+
+The amount of logging that should be output by the client can be set by specifying the log level in the logger. 
+The log is printed to the standard output, normally the terminal window from which the compiler is invoked.
```

Comment:
I think there should be a mention that we use the python standard logging and a link to https://docs.python.org/3.9/library/logging.html.